graylog2-web-interface/src/views/components/SearchResult.jsx

```diff
@@ -1,12 +1,11 @@
 // @flow strict
-import * as React from 'react';
+import React, { useContext } from 'react';
```

Comment:
We should split this up into:

```suggestion
import * as React from 'react';
import { useContext } from 'react';
```

This way we really do have `React` containing all exports and not just `default`.

Reply:
So far I've used `* as React` only when needed, but I see the benefit in always exporting not just `default`